content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
     {{< /caution >}}
 
 1.  Apply the CNI plugin of your choice:
-    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) 
     to install the CNI provider. Make sure the configuration corresponds to the Pod CIDR specified in the kubeadm configuration file if applicable.
-
-    In this example we are using Weave Net:
-
-    ```sh
-    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
-    ```
+  
+  {{<note>}}
+  You must pick your own CNI plugin and deploy it before you move on to next step.
+  If you don't do this, you will not be able to launch your cluster properly.
+  {{<note>}}
```

Comment:
this note might not be needed, but deferring to the SIG docs reviewers to see what they think.
the note is indeed true, but in `Apply the CNI plugin of your choice:` we already link to a step that is implicitly required.

Reply:
So, what shall I note in there?

Reply:
potentially, no extra note is needed.

Reply:
The indentation of this note doesn't look right to me.

I would leave the note in - people do overlook this detail.

Reply:
Try indenting the note to 4 spaces. It should match the indentation of other text in the list item.

Reply:
okay @sftim